docs/UserGuide.adoc

```diff
@@ -301,26 +301,30 @@ Examples:
 * `(rt)list` +
 Lists all restaurants.
 
-==== [Coming Soon] Find restaurant : `(rt)find`
+==== Search : `(rt)find`
 
-Finds restaurants based on a number of criteria. +
-Format: `(rt)find [k/KEYWORD] [l/LOCATION] [p/PRICE_POINT] [o/OPERATING_HOURS]`
+Searches for restaurants based on a number of criteria. +
+Format: `(rt)find [n/RESTAURANT_NAME] [l/LOCATION]`
 
 ****
 * At least one search criteria must be filled in.
-* There are 3 price points, each one distinguished according to the number of dollar signs. The lowest price point is `$`. The medium price point is `$$`. The highest price point is `$$$`.
+* All are case insensitive.
+* LOCATION must be exact matches (case insensitive).
```

Comment:
Do we still want to keep this condition? I think it will be better if we let the user input like say l/clementi then anything with l/clementi will show up regardless of whether its clementi east, clementi west?

Reply:
Eh sorry it's actually not like this. I'll delete the exact matches thing